src/applications/disability-benefits/all-claims/components/Autocomplete.jsx

```diff
@@ -9,6 +9,24 @@ const INSTRUCTIONS =
 
 const createFreeTextItem = val => `Enter your condition as "${val}"`;
 
+const isElementVisibleOnPage = element => {
+  const { top, bottom, left, right } = element?.getBoundingClientRect();
+
+  return (
+    top >= 0 &&
+    left >= 0 &&
+    bottom <= window.innerHeight &&
+    right <= window.innerWidth
+  );
+};
+
+const isElementVisibleInContainer = (element, container) => {
+  const { top, bottom } = element?.getBoundingClientRect();
+  const containerRect = container?.getBoundingClientRect();
+
+  return top >= containerRect.top && bottom <= containerRect.bottom;
+};
+
```

Comment:
Probably not perfect for our use case, but just checking that none of the other scroll-y helper functions in vets-website would work for this? Scroll/scrollto/scrolltotop/scrollIfFocusedAndNotInView etc

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/utilities/ui/scroll.js

https://github.com/department-of-veterans-affairs/vets-website/blob/ab9cdae449a9d5f8168d3184f2758734a1bbcedf/src/applications/mhv-secure-messaging/util/helpers.js#L428